kmm/positions/read_kmm.py

```diff
@@ -6,32 +6,35 @@
 
 @validate_arguments
 def read_kmm(path: Path):
-    return pd.read_csv(
-        path,
-        sep="\t",
-        encoding="latin1",
-        names=[
-            "centimeter",
-            "track_section",
-            "kilometer",
-            "meter",
-            "track_lane",
-            "1?",
-            "2?",
-            "3?",
-            "4?",
-            "5?",
-            "sweref99_tm_x",
-            "sweref99_tm_y",
-            "8?",
-            "9?",
-        ],
-        dtype=dict(
-            track_section=str,
-            kilometer=np.int32,
-            meter=np.int32,
-            track_lane=str,
-            sweref99_tm_x=np.float32,
-            sweref99_tm_y=np.float32,
-        ),
-    )
+    try:
+        return pd.read_csv(
+            path,
+            sep="\t",
+            encoding="latin1",
+            names=[
+                "centimeter",
+                "track_section",
+                "kilometer",
+                "meter",
+                "track_lane",
+                "1?",
+                "2?",
+                "3?",
+                "4?",
+                "5?",
+                "sweref99_tm_x",
+                "sweref99_tm_y",
+                "8?",
+                "9?",
+            ],
+            dtype=dict(
+                track_section=str,
+                kilometer=np.int32,
+                meter=np.int32,
+                track_lane=str,
+                sweref99_tm_x=np.float32,
+                sweref99_tm_y=np.float32,
+            ),
+        )
+    except Exception:
+        raise ValueError("Unable to parse kmm2 file, invalid csv.")
```

Comment:
```suggestion
        raise ValueError("Unable to parse kmm2 file, invalid csv.") from e
```

Reply:
to re raise the exception and give it a cause

```
ParseError: Too many columns specified
The above exception was the direct cause of the following exception:
ValueError: Unable to parse kmm2 file
```

Reply:
https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
:open_mouth: very nice, thanks for the tip